### ISSUE TITLE
documentation: build on Debian 12

### ISSUE THREAD

**Issue by x64-elf-sh42** [davidgiven/wordgrinder#265]
When building on debian, a fresh install, i ran into a few packages that are also needed.

pkg-config    // for the ./configure script - not sure if this used to be bundled by default. if you built glfw etc. from source you'd also need it but i didn't.

make            // this is plainly obvious from the instructions, but might add it to the list so you can copy paste all the stuff and just run 1 apt thing - perhaps build-essential would be more complete - i had a few tools already

xxd               // for make during luau building

suggested change to docs:

old:
If you have Debian/Ubuntu:
    ninja-build libncursesw5-dev zlib1g-dev libglfw3-dev
    
    
new:
If you have Debian/Ubuntu:
    ninja-build libncursesw5-dev zlib1g-dev libglfw3-dev build-essential pkg-config xxd

I realize for devs these are easy to deduce, but looking at how user friendly linux is becoming, its not only devs these days using it, and the wordgrinder is nice for novelists and other people who might not necessarily know how to read these build errors (but can follow your instructions by copy pasting).

**Comment by x64-elf-sh42** [comment]
little correction, debian 12 did not come with python, so i also installed python3 and python3-pil as i got the PIL not found error after getting past xxd. 

new:
If you have Debian/Ubuntu:
ninja-build libncursesw5-dev zlib1g-dev libglfw3-dev build-essential pkg-config xxd python3 python3-pil

after getting python3 and python3-pil in the mix, it all built nicely.

on make install it gives:
test -f bin/wordgrinder && cp bin/wordgrinder /usr/local/bin/wordgrinder
test -f bin/wordgrinder.1 && cp bin/wordgrinder.1 /usr/local/man/man1/wordgrinder.1
cp: cannot create regular file '/usr/local/man/man1/wordgrinder.1': No such file or directory
make: *** [Makefile:49: install] Error 1

this is not a big issue though, as the wordgrinder is installed ok. seems just the manpage is not. it might be debian changed something there on how to install manpages, but since the program is so simple and has all the bindings in there, its not really needed.

**Comment by railslancer** [comment]
Agreed. This change would have saved me a little time getting set up on Ubuntu 24.04.  Thanks.
